nltk/cluster/kmeans.py

```diff
@@ -198,7 +202,8 @@ def demo():
     print('Means:', clusterer.means())
     print()
 
-    vectors = [numpy.array(f) for f in [[3, 3], [1, 2], [4, 2], [4, 0], [2, 3], [3, 1]]]
+    vectors = [numpy.array(f) for f in [[3, 3], [1, 2], [
+        4, 2], [4, 0], [2, 3], [3, 1]]]
```

Comment:
Seriously?

Reply:
Beyond 80 chars is very subjective. But yeah, the auto-pep8 tool is making a bad change for this line ;P